presto-hive-hadoop2/pom.xml

```diff
@@ -17,6 +17,12 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>org.iq80.snappy</groupId>
+            <artifactId>snappy</artifactId>
+            <version>0.5</version>
+        </dependency>
+
```

Comment:
Instead of bringing in this dependency, have you tried adding this upgrade to the root pom dependency management?

Reply:
This was a transitive dependency coming from this module. I will try adding in the root pom as well. Thanks for the suggestion.